classes/autoptimizeToolbar.php

```diff
@@ -0,0 +1,127 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+class autoptimizeToolbar {
+
+	public function __construct()
+	{
+		// If Cache is not available we don't add the Autoptimize Toolbar
+		if( !autoptimizeCache::cacheavail() ) return;
+
+		// Load admin toolbar feature once WordPress, all plugins, and the theme are fully loaded and instantiated.
+		add_action( 'wp_loaded', array( $this, 'load_toolbar' ) );
+	}
+
+	public function load_toolbar()
+	{
+		// We check that the current user has the appropriate permissions
+		if( current_user_can( 'manage_options' ) )
```

Comment:
no extra options, cute puppies die because of too many options ;-)

no, seriously; I try to limit the amount of options, there's too many 
already. if people want they can always use the filters.

On 09-05-16 21:14, Pablo Custo wrote:
> In classes/autoptimizeToolbar.php
> <https://github.com/futtta/autoptimize/pull/41#discussion_r62555959>:
>
>> +
>> +class autoptimizeToolbar {
>> +
>> +	public function __construct()
>> +	{
>> +		// If Cache is not available we don't add the Autoptimize Toolbar
>> +		if( !autoptimizeCache::cacheavail() ) return;
>> +
>> +		// Load admin toolbar feature once WordPress, all plugins, and the theme are fully loaded and instantiated.
>> +		add_action( 'wp_loaded', array( $this, 'load_toolbar' ) );
>> +	}
>> +
>> +	public function load_toolbar()
>> +	{
>> +		// We check that the current user has the appropriate permissions
>> +		if( current_user_can( 'manage_options' ) )
>
> Thanks for the Tip.
> Later, in the next pull-request, you would like to incorporate some
> options in the plugin configuration page to make changes to these
> conditions through check boxes? ... as well as the ability to modify the
> maximum size of the cache... etc.. etc ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/futtta/autoptimize/pull/41/files/00bb2855b5700cbc4bf5b2bc1d4b5e365d8de86b#r62555959>
>